File: calc.py
def calculate(num1, num2, operation):
    try:
        if operation == '+':
            return num1 + num2
        elif operation == '-':
            return num1 - num2
        elif operation == '*':
            return num1 * num2
        elif operation == '/':
            if num2 == 0:
                return "You can't divide by zero!"
            return num1 / num2
        else:
            return "Unknown operator"
    except Exception as e:
        return f"There's an error: {e}"

print("Dumb Calculator")
num1 = float(input("What will be your first operand?: "))
num2 = float(input("And the second one?: "))
operation = input("Choose operator (+, -, *, /): ")

result = calculate(num1, num2, operation)
print(f"Result: {result}")

File: dev-core/calc.js
const readline = require('readline');

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

function calculate(num1, num2, operation) {
  switch (operation) {
    case '+':
      return num1 + num2;
    case '-':
      return num1 - num2;
    case '*':
      return num1 * num2;
    case '/':
      return num2 === 0 ? "Can't divide by zero" : num1 / num2;
    default:
      return "Unknown operation";
  }
}

// Ввод данных
rl.question("Enter first number: ", (firstInput) => {
  const num1 = parseFloat(firstInput);

  rl.question("Enter second number: ", (secondInput) => {
    const num2 = parseFloat(secondInput);

    rl.question("Enter operation (+, -, *, /): ", (operation) => {
      const result = calculate(num1, num2, operation);
      console.log(`Result: ${result}`);
      rl.close();
    });
  });
});